docker-compose.yml

```
@@ -113,6 +113,29 @@ services:
       - db
       - redis
 
+  worked_examples:
+    container_name: worked_examples
+    image: flowminder/flowkit-examples:${CONTAINER_TAG:-latest}
+    ports:
+      - ${WORKED_EXAMPLES_PORT:?Must set WORKED_EXAMPLES_PORT env var}:8888
+    tty: true
+    stdin_open: true
+    environment:
+      - FLOWDB_PORT=5432
+      - FLOWDB_HOST=${DOCKER_FLOWDB_HOST:?Must set DOCKER_FLOWDB_HOST env var}
+      - FLOWMACHINE_FLOWDB_USER=${FLOWMACHINE_FLOWDB_USER:?Must set FLOWMACHINE_FLOWDB_USER env var}
+      - FLOWMACHINE_FLOWDB_PASSWORD=${FLOWMACHINE_FLOWDB_PASSWORD:?Must set FLOWMACHINE_FLOWDB_PASSWORD env var}
+      - REDIS_HOST=query_locker
+      - REDIS_PORT=6379
+      - REDIS_PASSWORD=${REDIS_PASSWORD:?Must set REDIS_PASSWORD env var}
+      - FLOWAPI_URL="http://flowapi:9090"
```

This actually needs to be without the quotation marks:
```suggestion
      - FLOWAPI_URL=http://flowapi:9090
```
The reason is that otherwise the example notebooks will pick it up with an extra set of quotation marks:
```
>>> os.getenv("FLOWAPI_URL)
'"http://flowapi:9090"'
```
... and raise an error when talking to flowapi:
```
InvalidSchema: No connection adapters were found for '"http://flowapi:9090"/api/0/run'
```